site/src/utils/hasura/transformHelpers.js

```diff
@@ -0,0 +1,176 @@
+const compact = require("lodash/compact")
```

Comment:
To reduce duplication, we should replace the implementations in `utils/airtable.js` with these ones. You should be able to delete the original implementations, and inside of `airtable.js` use something like:

```js
export * from "./hasura/helpers.js"
```

Also, nitpick, but can we call this file `helpers.js`?